src/main/java/com/google/finapp/ArgsModule.java

```diff
@@ -91,6 +95,9 @@ String provideSpannerDatabaseId() {
 
     @Parameter(names = {"--spanner_database_id"})
     String spannerDatabaseId;
+
+    @Parameter(names = {"--spanner_dao_impl"})
```

Comment:
Can you file an issue to add description and required in these annotations? In particular here it would be good to document what are the available choices.